lib/Controllers/PersonsController.js

```diff
@@ -627,15 +627,17 @@ class PersonsController {
             visible_to: (input.visibleTo !== null) ? input.visibleTo : null,
         };
 
```

Comment:
Set these properties as undefined/null so the returned object wouldn't have duplicate properties in both camelCase and snake_case.
```suggestion
		input.ownerId = undefined;
        input.orgId = undefined;
        input.visibleTo = undefined;
```

Reply:
@RuTsEST done!!